gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapper.java

```diff
@@ -36,24 +49,163 @@ public class MedicationRequestMapper extends AbstractMapper<DomainResource> {
     private MedicationRequestPlanMapper medicationRequestPlanMapper;
     private MedicationStatementMapper medicationStatementMapper;
     private MedicationMapperContext medicationMapperContext;
+    private final IdGeneratorService idGeneratorService;
 
     private static final String PPRF = "PPRF";
     private static final String PRF = "PRF";
 
+    private static final String PRESCRIPTION_TYPE_EXTENSION_URL
+        = "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescriptionType-1";
+    private static final String PRESCRIPTION_TYPE_CODING_SYSTEM
+        = "https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescriptionType-1";
+
+
     public List<DomainResource> mapResources(RCMRMT030101UKEhrExtract ehrExtract, Patient patient, List<Encounter> encounters,
         String practiseCode) {
```

Comment:
I think this [practice](https://www.nhsinform.scot/care-support-and-rights/nhs-services/doctors/registering-with-a-gp-practice/)?

Reply:
It is indeed, it's all through the medicationMapper with bad spelling, as it's not on code I've created, I'll fix it in a different pr.